src/main/java/org/mitre/synthea/engine/Generator.java

```diff
@@ -214,8 +230,9 @@ public Person generatePerson(int index) {
    * @param personSeed
    *          Seed for the random person
    * @return generated Person
+ * @throws Throwable 
    */
-  public Person generatePerson(int index, long personSeed) {
+  public Person generatePerson(int index, long personSeed) throws Throwable {
```

Comment:
Generally it's not recommended to say "throws Throwable" because that doesn't really convey any information about what could go wrong. Can you make this a more specific Exception?    (That said, I do think in general we've been bad about exceptions and should do a global cleanup)